Core/Core/Planner/CalendarFilter/Model/Interactor/CalendarFilterCountLimit.swift

```diff
@@ -16,10 +16,18 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
-public enum CalendarFilterCountLimit: Int {
-    case base = 10
-    case extended = 20
-    case unlimited = 9999
+public enum CalendarFilterCountLimit: Equatable {
+    case base
+    case extended(Int)
+    case unlimited
+
+    public var rawValue: Int {
+        switch self {
+        case .base: return 10
+        case .extended(let value): return value
+        case .unlimited: return 9999
+        }
+    }
 }
 
 public extension Optional where Wrapped == AppEnvironment.App {
```

Comment:
I noticed that on this extension, if we have`isCalendarFilterLimitEnabled` always evaluated to false for Student app. I am wondering whether we need for Student or Parent apps to still worry about parsing this property.

Reply:
Not really, I also thought about this but left it as-is to save some time but it would be a nice future improvement.

Reply:
I created a follow-up task for this improvement.